modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -28,3 +30,17 @@ function stanford_profile_styles_page_attachments(array &$attachments) {
   }
 
 }
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function stanford_profile_styles_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+
+  // Add the admin CSS library.
+  $element['#attached']['library'][] = 'stanford_profile_styles/admin';
```

Comment:
we don't need this here. the global admin library would get added via a `hook_page_attachments()`

Reply:
Yes, please move this attachment to hook_page_attachments and ensure that it is only added to the admin theme.